### ISSUE TITLE
Clarify that a N/A result is a pass.

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag-act#160]
In WCAG 2 it was important that N/A results meant that the page passed the SC. If at the test leverl there are N/A results then it should be made clear when the rules are aggregated against SCs that they cause a passing condition, not an N/A situation for the SC.

**Comment by WilcoFiers** [comment]
Thanks for the feedback David. I see no reason why we wouldn't include this. The comment came slightly too late to include in this draft, but if it's already with you, we'll take this as the first comment on this new draft.

**Comment by skotkjerra** [comment]
ACT rules are designed to map against a specific accessibility requirement. This CAN be a WCAG success criterion, but the ACT rules can also be mapped to other kinds of accessibility requirements - other than those specified in WCAG. Therefore, to keep the ACT rules format agnostic to the specific requirements we propose that only a note is inserted specifically relating to the translation of a N/A result to the passing of a WCAG success criterion.

**Comment by WilcoFiers** [comment]
@skotkjerra Remember to e-mail me the proposal.
